scraper/scraper.go

```diff
@@ -110,12 +110,15 @@ func scrapeInstances(addresses []string, endpoint string, processor Processor) <
 func store(serviceName string, r *repository.ErrorsRepository, m errorAggregateMap) {
 	errors := make([]repository.ErrorAggregate, 0, len(m))
 	for _, value := range m {
+		severity := severityWithFallback(value.Severity)
 		errors = append(errors, repository.ErrorAggregate{
 			AggregationKey: value.AggregationKey,
-			Severity:       severityWithFallback(value.Severity),
+			Severity:       severity,
 			TotalCount:     value.TotalCount,
 			LatestErrors:   toRepositoryErrorsWithContent(value.LatestErrors),
 		})
+		metrics.ScrappedAggregatedErrorTotal.WithLabelValues(serviceName, severity,
+			value.AggregationKey).Set(float64(value.TotalCount))
```

Comment:
Does it make sense to record the number of occurrences per instance instead? That makes the queries a little more complicated but it's easier to understand if one error is specific to an instance or not.

Reply:
Thats a good idea, but requires changing a lit bit the structure. I going to create a ticket to track that.